portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,9 +61,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Query commentsQuery = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    PreparedQuery results = datastore.prepare(commentsQuery);
+    List<Entity> results = datastore.prepare(commentsQuery).asList(FetchOptions.Builder.withLimit(2));
 
```

Comment:
make the limit a java constant at the top of your class

private static final int MAX_COMMENTS_FETCHED = 2;

Reply:
+1